ndcube/visualization/mpl_sequence_plotter.py

```diff
@@ -0,0 +1,131 @@
+from astropy.wcs.wcsapi import BaseLowLevelWCS
+
+try:
+    from sunpy.visualization.animator import ArrayAnimatorWCS
+except ImportError:
+    raise ImportError(
+        "Sunpy is required to animate NDCubeSequences. "
+        "Either install sunpy or extract data from sequence and visualize manually.")
+
+from .base import BasePlotter
+from .plotting_utils import prep_plot_kwargs
+
+__all__ = ['MatplotlibSequencePlotter']
+
+
+class MatplotlibSequencePlotter(BasePlotter):
```

Comment:
Rather than adding this class, you could dispatch on the type of `self._ndcube` in the `MatplotlibPlotter`? Or is this a prelude to more custom sequence specific plotting in the future?

Reply:
Yes, I have already have local implementations for creating a 2-D image and 1-D line plot in the cases where we have a sequence of 1-D cubes and a sequence of shape `(1,)` cubes.  But I was planning on submitting them after 2.0 is released.